yoti_python_sdk/dynamic_sharing_service/extension/location_constraint_extension_builder.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+
+class LocationConstraintExtensionBuilder(object):
```

Comment:
Should this extend the ExtensionBuilder rather than being its own standalone class?

```suggestion
from .extension_builder import ExtensionBuilder

class LocationConstraintExtensionBuilder(ExtensionBuilder):
```

Reply:
I... would say no as that would allow people to change the ExtensionType on a LocationConstraintExtension. Which is wrong.

It's more of an "Has-a"/aggregation than an "Is-a" inheritance relationship.

Reply:
I think Alex was just asking because isn't that what the [TransactionalFlowExtensionBuilder](https://github.com/getyoti/yoti-python-sdk/pull/71/files#diff-6e8884471a7d49961ae6e1d004a72fe5) is doing?

Reply:
Yeah. I wasn't super happy about that, but TransactionalFlow needed the same methods as Extension. I think Extension and TransactionalFlowExtension are... less concrete/well defined? Compared to LocationConstraint.

Reply:
I think we should probably be consistent and have either both of them or none of them using ExtensionBuilder. I'll leave it up to you which one you prefer 👍